AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -9,9 +9,11 @@
 from skimage import transform
 from skimage.filters import gaussian
 from sys import platform as _platform
-if _platform == "darwin": # Mac OSX
+
+if _platform == "darwin":  # Mac OSX
     import matplotlib as mpl
-    mpl.use('TkAgg')
+
+    mpl.use("TkAgg")
 import matplotlib.pyplot as plt
```

Comment:
isn't the idea to avoid importing pyplot?

Reply:
in https://python4astronomers.github.io/plotting/advanced.html, `pyplot` is still used to create the figure, i.e. `fig=plt.figure`.
They might have ways to avoid importing it but in order to go OO the necessary step where to use `subplots` and `axes`.

Reply:
@maf88 you're right-- my bad

Reply:
Apperently it's not that complicated to go pure OO: https://matplotlib.org/gallery/api/agg_oo_sgskip.html. 

Yet, we will loose the ability to display in a window and other small things. It's important to know that I found only 4 .py files that are importing `matplotlib` and their methods were not use anywhere. It seems like tools that we keep just in case, or that were used.

If instead of showing I can save them, then we can remove pyplot.

Reply:
good-- we don't need interactive show, so go ahead and remove pyplot